src/report/context.rs

```diff
@@ -0,0 +1,54 @@
+use report::{Report, BlockReport};
+
+/// `ContextReport` holds the results of a context's test execution.
+#[derive(PartialEq, Eq, Clone, Default, Debug)]
+pub struct ContextReport {
+    sub_reports: Vec<BlockReport>,
+}
+
+impl ContextReport {
+    pub fn new(sub_reports: Vec<BlockReport>) -> Self {
+        ContextReport { sub_reports: sub_reports }
+    }
+
+    pub fn get_blocks(&self) -> &[BlockReport] {
+        &self.sub_reports[..]
+    }
+}
+
+impl Report for ContextReport {
```

Comment:
tests would be nice here

Reply:
I prototyped a bit, it needs a lot of setup code...
I think it would be nice to make an evolution of `BlockReport`, `ContextReport`, and `ExampleReport` to have fixtures generators.

I would suggest something like:
```rust
// arrange
let report1 = BlockReport::fixture_from(ExampleReport::fixture_failed());
let report2 = BlockReport::fixture_from(ExampleReport::fixture_success());
let context_report = ContextReport::new(vec!(report1, report2));
// assert
assert_eq!(false, context_report.is_success())
```
With all the `fixture_*` functions members that are gated by a `#[cfg(test)]`

That's only a suggestion, ofc